docs/project/2024/gabriel.danaila/index.md

```diff
@@ -0,0 +1,73 @@
+# DIY Pico Modulated Beeps to Tesla Coil
+This Tesla Coil has been modified so that the spark "sings" melody generated by a Raspberry Pico Microcontroller.
+
+:::info 
+
+**Author**: Gabriel Dănăilă \
+**GitHub Project Link**: https://github.com/UPB-FILS-MA/project-Gabi10017
+
+:::
+
+## Description
+
+The audio beeps are generated on the microcontroller which uses PWM in order to send a digital signal to a DAC which converts the signal into an analog one and then sends it to the Tesla Coil's PCB which modulates it into electrical voltage which is sent to the coil, playing a sound. The beep's pitch and volume can be controlled using 2 potentiometers connected using ADC.
+
+## Motivation
+
+I chose this project because I think it is an interesting, innovative idea to put in practice, that is neither too easy or too complex to do for a single-person project. This project idea was brought to me by the lab teacher.
+
+## Architecture 
+
+Add here the schematics with the architecture of your project. Make sure to include:
+ - what are the main components (architecture components, not hardware components)
+ - how they connect with each other
+
```

Comment:
Please add a block diagram of your project.

Reply:
Added.